service-registry/src/main/java/org/apache/servicecomb/serviceregistry/task/MicroserviceInstanceRegisterTask.java

```diff
@@ -68,6 +68,10 @@ protected boolean doRegister() {
     microserviceInstance.getHealthCheck().setTimes(serviceRegistryConfig.getResendHeartBeatTimes());
 
     String instanceId = srClient.registerMicroserviceInstance(microserviceInstance);
+    MicroserviceInstance temIns = srClient.findServiceInstance(microserviceInstance.getServiceId(),instanceId);
+    if(temIns!=null){
+      microserviceInstance.setTimestamp(temIns.getTimestamp());
```

Comment:
Does service-center already add timestamp property and which version？ And if it is neccessary to set timestamp information for the service itself (may be do not need set this property, or we'd better to keep itself reference from service center, not set a single property)?

Reply:
i remember SC provide this property in very very very old version
but better to confirm that.

Reply:
it's always provide this property , java-chassis don't use it. we need to display the timestemp